packages/trader/src/App/Components/Elements/Modals/ServicesErrorModal/services-error-modal.tsx

```diff
@@ -56,6 +58,9 @@ const ServicesErrorModal = ({
         case 'CompanyWideLimitExceeded':
             return <CompanyWideLimitExceededModal is_visible={is_visible} onConfirm={onConfirm} />;
         case 'PleaseAuthenticate':
+            if (multipliers_account_status === 'pending') {
```

Comment:
This condition is very specific to a certain condition and needs to be removed from a shared component such as this file. please share your thoughts @amina-deriv

Reply:
Thanks @matin-deriv for the suggestion. I moved it from service error modal. Now, when the purchase buttons are clicked, we verify if the account is  MF and if its status is pending. If both conditions are met, the modal is shown. Can you help to review his?